frontend/src/components/new-car-card/new-car-card.tsx

```diff
@@ -1,35 +1,100 @@
-import { FC } from 'react';
+import React from 'react';
 
-import bmw from '@assets/images/bmw_logo.svg';
 import compare from '@assets/images/compare.svg';
-import heart from '@assets/images/heart.svg';
-import new_car_1 from '@assets/images/mock_car_picture.png';
+import { WishlistInput } from '@autoline/shared/common/types/types';
+import { ExtendedCarCardPropsType } from '@common/types/types';
+import { Heart } from '@components/common/icons/icons';
+import { clsx } from 'clsx';
 
 import styles from './styles.module.scss';
 
-const NewCarCard: FC = () => {
+const NewCarCard: React.FC<ExtendedCarCardPropsType> = (props) => {
```

Comment:
Please use props destructuring at the start of the component: `const {...} = props`.
It'll help to avoid contestant `props.` repetition